Resources/Prototypes/Entities/Structures/cargo_telepad.yml

```diff
@@ -47,3 +47,5 @@
     board: CargoTelepadMachineCircuitboard
   - type: Appearance
   - type: CollideOnAnchor
+  - type: NameIdentifier
+    group: CargoTelepads
```

Comment:
dvir what happened to the EOF newline

Reply:
https://github.com/space-wizards/space-station-14/blob/cf96679d0bcfe65061d0d220af15575832423425/Resources/Prototypes/Entities/Structures/cargo_telepad.yml#L4